src/main/java/frc/robot/commands/FloorIntakeCommands/FI_IntakeForward.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.commands.FloorIntakeCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.FloorIntakeSubsystem;
+
+public class FI_IntakeForward extends Command{
+    FloorIntakeSubsystem floorIntake;
+ 
+
+    public FI_IntakeForward(FloorIntakeSubsystem floorIntake){
+        this.floorIntake = floorIntake;
+    }
+
+    public void execute(){ 
```

Comment:
Note that `execute` is also defined in the `Command` class, so `@Override` can/should also be used here.